plugin.xml

```diff
@@ -22,17 +22,36 @@
         </config-file>
 
         <framework src="com.github.shogo4405.HaishinKit~kt:haishinkit:0.10.4" />
-        <framework src="src/android/build-extras.gradle" custom="true" type="gradleReference" />
+      <!--  <framework src="src/android/build-extras.gradle" custom="true" type="gradleReference" />-->
 
     </platform>
 
     <platform name="ios">
-        <config-file parent="/" target="config.xml">
+        <config-file parent="/*" target="config.xml">
             <feature name="IoniCordovaRTMPandHLS">
                 <param name="ios-package" value="IoniCordovaRTMPandHLS" />
             </feature>
         </config-file>
-
-        <source-file src="src/ios/IoniCordovaRTMPandHLS.m" />
+        <!-- 
```

Comment:
Same here :)

Reply:
@ediboy right, so regarding this one, is because for cordova CLI <= 9.0.1 (our TKA app uses this version) in order to add library dependency in the podspec file we need to use the the instruction in the line 45, but for cordova CLI > 9.0.1 (Sample Project uses cordova CLI 10), we need to use the intruction in the commented code.

Having said that, we will need to provide support for both, once we have everything merge, we can create different releases in order to support different cordova CLIs.

For now lets leave this